pkg/network/usm/monitor_tls_test.go

```diff
@@ -561,6 +561,69 @@ func TestHTTPSGoTLSAttachProbesOnContainer(t *testing.T) {
 	})
 }
 
+func TestOldConnectionRegression(t *testing.T) {
+	modes := []ebpftest.BuildMode{ebpftest.RuntimeCompiled, ebpftest.CORE}
+	ebpftest.TestBuildModes(t, modes, "", func(t *testing.T) {
```

Comment:
Right now, this will only benefit GoTLS because when we miss the OpenSSL initialization we don't have the PID/FD either (just the SSL context pointer);

Having said that, this change could benefit other encryption libraries or tracing use-cases. For example, if we were to instrument Read/Write syscalls, this change could come handy.